packages/openactive-integration-tests/test/helpers/feature-helper.js

```diff
@@ -49,6 +49,9 @@ const { BOOKABLE_OPPORTUNITY_TYPES_IN_SCOPE, IMPLEMENTED_FEATURES, AUTHENTICATIO
  *   Defaults to 1.
  * @property {string[]} [skipOpportunityTypes] Some tests (eg access-channel tests for virtual events) only apply to
  *   certain types of opportunity (in the example provided, access-channel tests should not be run for facility slots)
+ * @property {boolean} [supportsApproval] TEMPORARY field until approval works for all tests. Approval will only be
+ *   attempted for a test that has this field set to true (unless skipBookingFlows is set).
```

Comment:
I decided to implement this due to how many tests were not working with approval. This way, we can have config set to try both approval & simple tests and it will only check both for tests that have this field set to true